### DISCUSSION TITLE
URL change for dashboard - after it created

### DISCUSSION THREAD

**Discussion by Masterz69** [home-assistant/frontend#8642] [Dashboard]
Would be nice to have ability to change dashboard URL.

Can enter it on creation:
![image](https://user-images.githubusercontent.com/15225823/111069651-bd603a00-84d6-11eb-97b9-80a85fd4416c.png)

But can't change it later one:
![image](https://user-images.githubusercontent.com/15225823/111069666-cf41dd00-84d6-11eb-9ade-8d7e2f73af75.png)

:(

**Comment by glasslike** [comment]
I'm totally with you! I created the dashboard "test" and fill it with many things and cannot change the URL! Sad.

**Comment by charredchar** [comment]
I don't get how this is not a thing yet... You can change the URL of tabs within a dashboard but I can't find anything to change the URL of the dashboard itself unless you created the entire thing in your configuration.yaml.

**Comment by KTibow** [comment]
This would require renaming the `lovelace.your-url` file in `.storage`, changing the `"key"` inside of it, and updating where it's referenced in `lovelace_dashboards`.

**Comment by trichromic-ab** [comment]
Adding my vote here.
Please can we have a way to edit a dashboard URL after its created, just like we can with View tab URLs?

**Comment by Xitee1** [comment]
As a workaround I always switch to YAML editor and copy-paste the content to the new dashboard and then switch it back to graphical.

**Reply by Bracco66** [reply]
It works for view and card, not for dashboard

**Reply by Xitee1** [reply]
Why not? Edit the dashboard by clicking the menu in the top right corner and copy all the views, then create a new dashboard, edit again and paste.

**Reply by Bracco66** [reply]
Sorry. I didn't realize you were creating a new dashboard. It's actually a bit convoluted but it works.

**Comment by pickonedev** [comment]
+1

**Comment by treecamp** [comment]
+1

**Comment by kenwiesner** [comment]
+1

**Comment by rwjack** [comment]
+1

**Comment by Extreme79** [comment]
I'm searching for an easy (and safe!) way to do that. Me too!

**Comment by RaBuAc** [comment]
Me too!

**Comment by Loic691** [comment]
same ;-)

**Comment by karwosts** [comment]
The reason this is not a thing in frontend is because core does not support an API to change this. Without that API there's nothing we can do from here. 

Someone would need to propose a core feature to support updating the URL after creating, then we could add it.

**Comment by Hadatko** [comment]
+1

**Comment by actstorms** [comment]
+1

**Comment by Bracco66** [comment]
+1

**Comment by technoo10201** [comment]
+1

**Reply by KTibow** [reply]
Did you know that every time you say "+1" 18 people get pinged?
Did you know that nobody will count your "+1"s?
Did you know that you would make a larger impact if you just hit the upvote button on this discussion?

**Reply by Bracco66** [reply]
I didn't know it.
thanks for the information
